assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml

```diff
@@ -1,28 +1,28 @@
 type: evaluationresult
 name: truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering
-version: 2.19.040
-display_name: truthfulqa_generation_mistralai-Mixtral-8x7B-v01_question_answering
-description: mistralai-Mixtral-8x7B-v01 run for truthfulqa_generation dataset
+version: 2.23.05
+display_name: mistralai-Mixtral-8x7B-v01_truthfulqa_generation_text_generation
+description: mistralai-mixtral-8x7b-v01  for truthfulqa_generation dataset
```

Comment:
May I know why were the `display_name` and `description` changed? This shouldn't have happened. Please verify for all the assets.